charts/ceph-csi-cephfs/values.yaml

```diff
@@ -275,6 +275,26 @@ secret:
   adminID: <plaintext ID>
   adminKey: <Ceph auth key corresponding to ID above>
 
+# This is a sample configmap that helps define a Ceph configuration as required
+# by the CSI plugins.
+# Sample ceph.conf available at
+# https://github.com/ceph/ceph/blob/master/src/sample.ceph.conf Detailed
+# documentation is available at
+# https://docs.ceph.com/en/latest/rados/configuration/ceph-conf/
+cephconf: |
+  [global]
+    auth_cluster_required = cephx
+    auth_service_required = cephx
+    auth_client_required = cephx
+
+    # Workaround for http://tracker.ceph.com/issues/23446
+    fuse_set_user_groups = false
```

Comment:
just wondering this is still valid , the referenced issue has been fixed already ? @kotreshr any thoughts here?